bin/datahelix.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
```

Comment:
This would disable echo for the rest of the CMD session.

I suggest instead we prefix each command with a `@` which disables `echo` for that command only.